Ship_Game/Ships/Ship_Initialize.cs

```diff
@@ -330,9 +330,13 @@ public static Ship CreateShipNearPlanet(UniverseState us, string shipName, Empir
         {
             float randomRadius = owner.Random.Float(p.Radius - 100, p.Radius + 100);
             Ship ship = CreateShipAt(us, shipName, owner, p, p.Position.GenerateRandomPointOnCircle(randomRadius, owner.Random), doOrbit);
-            if (initLaunch && ship != null && !ship.IsPlatformOrStation)
+            if (initLaunch && ship != null && !ship.IsPlatformOrStation && !(ship.MaxFTLSpeed == 0 || ship.MaxSTLSpeed == 0))
                 ship.InitLaunch(LaunchPlan.Planet, p);
-
+            else if (initLaunch && ship != null && (ship.MaxFTLSpeed == 0 || ship.MaxSTLSpeed == 0))
+            {
+                ship.Position += 2;

```

Comment:
What is this for ?

Reply:
to make starting position not straight above planet

Reply:
you can do it like this:
```
                Ship createdStartingShip = Ship.CreateShipNearPlanet(UState, starterShip, empire, homePlanet, true);
                if (createdStartingShip != null && (createdStartingShip.MaxFTLSpeed == 0 || createdStartingShip.MaxSTLSpeed == 0))
                {
                    createdStartingShip.Position = homePlanet.Position.GenerateRandomPointOnCircle(1000 + homePlanet.Radius, UState.Random);
                    createdStartingShip.TetherToPlanet(homePlanet);

                }```